plugins/auto-sizes/includes/improve-calculate-sizes.php

```diff
@@ -232,15 +240,15 @@ function auto_sizes_get_layout_width( string $alignment ): string {
 	return $layout_widths[ $alignment ] ?? '';
 }
 
-/**
- * Filters the context keys that a block type uses.
- *
- * @since n.e.x.t
- *
- * @param string[]      $uses_context Array of registered uses context for a block type.
- * @param WP_Block_Type $block_type   The full block type object.
- * @return string[] The filtered context keys used by the block type.
- */
+	/**
+	 * Filters the context keys that a block type uses.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param string[]      $uses_context Array of registered uses context for a block type.
+	 * @param WP_Block_Type $block_type   The full block type object.
+	 * @return string[] The filtered context keys used by the block type.
+	 */
```

Comment:
```suggestion
/**
 * Filters the context keys that a block type uses.
 *
 * @since n.e.x.t
 *
 * @param string[]      $uses_context Array of registered uses context for a block type.
 * @param WP_Block_Type $block_type   The full block type object.
 * @return string[] The filtered context keys used by the block type.
 */
```